src/main/java/co/omise/models/Link.java

```diff
@@ -152,50 +231,47 @@ public CreateRequestBuilder multiple(boolean multiple) {
             this.multiple = multiple;
             return this;
         }
-    }
 
-    /**
-     * The {@link RequestBuilder} class for retrieving a particular Link.
-     */
-    public static class GetRequestBuilder extends RequestBuilder<Link> {
-        private String linkId;
-
-        public GetRequestBuilder(String linkId) {
-            this.linkId = linkId;
-        }
-
-        @Override
-        protected HttpUrl path() {
-            return buildUrl(Endpoint.API, "links", linkId);
+        public CreateRequestBuilder title(String title) {
+            this.title = title;
+            return this;
         }
 
         @Override
-        protected ResponseType<Link> type() {
-            return new ResponseType<>(Link.class);
+        protected RequestBody payload() throws IOException  {
+            return serialize();
         }
     }
 
-    /**
-     * The {@link RequestBuilder} class for retrieving all Links that belong to an account.
-     */
-    public static class ListRequestBuilder extends RequestBuilder<ScopedList<Link>> {
+    public static class ListChargesRequestBuilder extends RequestBuilder<ScopedList<Charge>> {
```

Comment:
There are so many different RequestBuilders in this class. Maybe it wouldn't be a bad idea to put a description on what each of them represent.